src/pykeen/experiments/cli.py

```diff
@@ -160,16 +161,17 @@ def _help_reproduce(
         move_to_cpu=move_to_cpu,
         save_replicates=save_replicates,
     )
-    shutil.copyfile(path, os.path.join(output_directory, 'configuration_copied.json'))
+    shutil.copyfile(path, output_directory / 'configuration_copied.json')
 
 
 @experiments.command()
 @click.argument('path')
```

Comment:
you can add `type=pathlib.Path` as a kwarg to `click.argument('path', type=pathlib.Path)` to make this work the way you want